docs/src/references/agent/tedge-configuration-management.md

```diff
@@ -4,44 +4,43 @@ tags: [Reference, Configuration]
 sidebar_position: 6
 ---
 
-# Configuration Management Plugin
+# Configuration Management
 
-Thin-edge provides an operation plugin to manage device configuration files.
+Thin-edge implements an operation to manage device configuration files.
 
+* The __tedge-agent__ enables configuration management on its running device
+  and, combined with a cloud mapper, extends this capability to enable configuration management from the cloud.
 * This management is bi-directional:
   * A device can act as a reference,
     with all the managed files being uploaded to the [tedge file transfer repository](../tedge-file-transfer-service.md)
     and stored there as a configuration snapshot.
   * A configuration update can be pushed from the tedge file transfer repository to any devices of the same type,
     i.e. supporting the same kind of configuration files.
-  * This plugin, combined with a cloud mapper, enables configuration management from the cloud.
-* With this operation plugin, the device owner defines the list of files
+* On each device running the agent, the device owner defines the list of files to be managed
   (usually configuration files, but not necessarily).
-* The plugin configuration itself can be managed both locally and from the cloud,
+* The configuration of this feature itself can be managed both locally and from the cloud,
   meaning, the device owner can update the list of files to be managed from the cloud using the cloud mapper.
 * The configuration files are managed according to their type,
   a name chosen by the device owner to categorize each configuration.
   By default, the full path of a configuration file on the device is used as its type.
-* When files are downloaded from the tedge file transfer repository to the device where the plugin is installed,
+* When files are downloaded from the tedge file transfer repository to the a target child device,
```

Comment:
```suggestion
* When files are downloaded from the tedge file transfer repository to the target device,
```
Since this behaviour applies to the main device as well.